packages/app/src/runs/RunsSkeleton.vue

```diff
@@ -1,43 +1,67 @@
 <template>
-  <div
-    data-cy="runs-loader"
+  <ul
+    v-if="isUsingGit"
+    data-cy="runsSkeleton-git"
+    class="flex flex-col mb-[24px] gap-[16px] relative before:content-[''] before:absolute before:top-[20px] before:bottom-[10px] before:w-[2px] before:border-2 before:border-dashed before:border-l-0 before:border-y-0 before:border-r-gray-100 before:left-[7px]"
   >
-    <div
-      v-for="i in numberOfLines"
+    <li
+      v-for="i in numberOfLinesGit"
       :key="i"
-      class="mb-4"
     >
-      <ListRowHeader
-        disabled
-        class="border border-gray-100"
+      <div
+        class="flex items-center my-[10px] gap-[8px] children:h-[8px]"
```

Comment:
```suggestion
        class="flex items-center my-[10px] gap-[8px] children:h-[14px]"
```
Increase height of children to better match real components